dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/dispatch/executor/NettyExecutorManager.java

```diff
@@ -101,11 +102,9 @@ public Boolean execute(ExecutionContext context) throws ExecuteException {
         Command command = context.getCommand();
         // execute task host
         Host host = context.getHost();
-        boolean success = false;
-        while (!success) {
+        while (true) {
```

Comment:
It seems never return if no exception?

Reply:
It will loop all nodes, if all nodes failed, it will throw exception.